app/livechat/client/voip.ts

```diff
@@ -0,0 +1,72 @@
+import moment from 'moment';
+
+import { MessageTypes } from '../../ui-utils/client';
+import { IVoipMessage } from '../../../definition/IMessage';
+
+type IMessageFuncReturn = {
+	comment?: string;
```

Comment:
```suggestion
```
Comment is never used

Reply:
Removed it. @KevLehman I am not sure if this is the right place for this file. I feel that from design perspective, voip shouldn't be associated with livechat. There exists a livechat call which is purely webRTC based (No SIP, no mobile calling involved). Will this file create a conflict with this functionality? 

Furthermore, as I learnt recently, we are moving away from putting things in app folder. I am not sure if it applies to this voip file. Can you take a look?